docs/how-to/uc-boot-parameters.rst

```diff
@@ -0,0 +1,89 @@
+How to configure boot parameters on Ubuntu Core 
+================================================
+
+.. note::
+
+    This guide assumes access to features available exclusively to `dedicated Snap Store`_ users.
+
+Ubuntu Core doesn't have the :code:`/etc/default/grub` file mapped for the kernel parameters, nor the `update-grub`_ utility.
+On Ubuntu Core, the kernel parameters can be modified statically or dynamically.
+Static configurations are added during the image build.
+On the other hand, dynamic configurations are set `using snap options`_ on a running system.
+This guide focuses on the dynamic configuration method, suitable for iteratively tuning the operating system.
+
+.. hint::
+    
+    Refer to :doc:`../how-to/uc-image-creation` if you want to create a real-time Ubuntu Core image.
+
+Open a terminal and access your real-time Ubuntu Core instance.
+The instance should be on a physical device rather than a virtual machine.
+While a virtual machine can be used to follow the steps in this guide, it will not provide the benefits of a real-time kernel.
+
+There are two snap options for setting the kernel parameters:
+
+- `system.kernel.cmdline-append`_
+- `system.kernel.dangerous-cmdline-append`_
+
+The first one is used to append permitted kernel boot parameters that are verified against an `allow list`_ in the `gadget snap`_. 
```

Comment:
Maybe move this as a 'description' directly under the bullet point for each snap option?
E.g.

- `system.kernel.cmdline-append` - Used to append permitted kernel boot parameters that are verified...
- `system.kernel.dangerous-cmdline-append` - Used to append any other...

So the context is present instead of being referred to as "first" or "second one"?

Thanks.